lib/MainTemplate.js

```diff
@@ -30,8 +30,7 @@ module.exports = class MainTemplate extends Template {
 			const buf = [];
 			if(chunk.entryModule) {
 				buf.push("// Load entry module and return exports");
-				buf.push("return " + this.renderRequireFunctionForModule(hash, chunk, JSON.stringify(chunk.entryModule.id)) +
-					"(" + this.requireFn + ".s = " + JSON.stringify(chunk.entryModule.id) + ");");
+				buf.push(`return ${this.renderRequireFunctionForModule(hash, chunk, JSON.stringify(chunk.entryModule.id))}(${this.requireFn}.s = "${JSON.stringify(chunk.entryModule.id)}");`);
```

Comment:
`"${JSON.stringify(chunk.entryModule.id)}"`

i think these `"` should be omitted